lips/lip-2.md

```diff
@@ -0,0 +1,119 @@
+|ValidatorOperator                                                                                                                                   |LibraRoot                   |N                                                                                                      |-             |N                   |-                                                                                                                                                                                         |Y         |2                                                                                                                                         |
+|DesignatedDealer                                                                                                                               |LibraRoot                   |N                                                                                                      |-             |Y                   |N                                                                                                                                                                                         |Y         |2                                                                                                                                         |
+|ParentVASP                                                                                                                                          |LibraRoot                   |Per VASP                                                                                               |-             |Y                   |Y                                                                                                                                                                                         |Y         |-                                                                                                                                         |
+|ChildVASP(**parent**)                                                                                                                                   |ParentVASP                  |N                                                                                                      |-             |Y                   |Y                                                                                                                                                                                         |Y         |-                                                                                                                                         |
+
+* LibraRoot - The root authority of Libra. Controlled jointly by Libra Payment Networks and the Association Council.
```

Comment:
I would leave out organizational and control comments. Instead, say something like:  An entity responsible for creating top-level accounts and performing actions that reconfigure the blockchain.

Reply:
I think the organizational and control spec is important here. A role is supposed to describe *who* an entity is and permissions are supposed to say *what* an entity can do. Saying "LibraRoot can create top-level accounts and trigger reconfig" is important, but should be expressed in permissions rather than in a description of the role.

Reply:
Got it. So let's see if it gets flagged by legal. As a fallback, we could perhaps say: LibraRoot is the root authority of the Libra payment system.